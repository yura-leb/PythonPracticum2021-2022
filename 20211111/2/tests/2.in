(1,1), (2,2), ('a', 23)
(1,2), (6,2), (4.2,5)