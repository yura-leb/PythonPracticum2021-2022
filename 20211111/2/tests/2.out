Invalid Input
7.50
